src/usr/local/www/widgets/widgets/system_information.widget.php

```diff
@@ -60,17 +60,14 @@
 	'memory_usage' => array('itemtitle' => gettext('Memory Usage'), 'category' => 'resource_usage', 'itemsort' => 19),
 	'swap_usage' => array('itemtitle' => gettext('Swap Usage'), 'category' => 'resource_usage', 'itemsort' => 20),
 	'disk_usage' => array('itemtitle' => gettext('Disk Usage'), 'category' => 'resource_usage', 'itemsort' => 21),
-	'admin_access_methods' => array('itemtitle' => gettext('Admin Lockdown'), 'category' => 'security', 'itemsort' => 22),
-	'admin_users' => array('itemtitle' => gettext('Users with Admin Access'), 'category' => 'security', 'itemsort' => 23)
```

Comment:
Removed these new items that are not implemented yet. @stilez You can easily add these back when you implement some code for them.